packages/contracts/contracts/newsroom/NewsroomFactory.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity 0.4.19;
+// TODO(ritave): Think of a way to not require contracts out of package
+import "../multisig/Factory.sol";
+
+import "../interfaces/IMultiSigWalletFactory.sol";
+import "./Newsroom.sol";
+
+contract NewsroomFactory is Factory {
+  IMultiSigWalletFactory public multisigFactory;
+
+  function NewsroomFactory(address _multisigFactory) public {
+    multisigFactory = IMultiSigWalletFactory(_multisigFactory);
+  }
+
+  function create(address[] _owners, uint _required)
+    public
+    returns (Newsroom newsroom)
+  {
+    address wallet = multisigFactory.create(_owners, _required);
+    newsroom = new Newsroom();
+    newsroom.transferOwnership(wallet);
```

Comment:
we could just modify the newsroom contract to take an address in the constructor, I guess it doesn't matter too much though

Reply:
`Ownable.sol` doesn't support taking address in constructor, might as well leave it explicit.